spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb

```diff
@@ -157,59 +157,59 @@
       wizardstore["degree_options"] = TeacherTrainingAdviser::Steps::HaveADegree::DEGREE_OPTIONS[:studying]
     end
 
-    it "returns current calendar year + 2 years if first year and before 4th September" do
+    it "returns current calendar year + 2 years if first year ends before 17th September" do
```

Comment:
@sarahcrack @spencerldixon I don't think this was a typo?

ie `first year` refers to the degree status option chosen (similarly `second year`, `final year`, `other` and doesn't refer to the date?

Reply:
ah that makes more sense, it just didn't read right while we were working on them. Yes let's change them back

Reply:
@gemmadallmandfe @spencerldixon I have amended the test wording and raised a new PR: https://github.com/DFE-Digital/get-into-teaching-app/pull/4172.